ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -269,7 +285,18 @@ export default defineComponent({
       reviewed: false,
       isEditingMode: computed((): boolean => localState.currentlyEditingHomeOwnerId >= 0),
       isAddingMode: computed((): boolean => props.isAdding),
-      showTableError: computed((): boolean => showGroups.value && (hasMinimumGroups() || hasEmptyGroup.value)),
+      showTableError: computed((): boolean => {
+        return showGroups.value &&
+          (
+            props.validateTransfer &&
```

Comment:
Would still work for Registrations as expected?

Reply:
props.validateTransfer being false in a registration might give an incorrect error...

Reply:
Good catch guys, updating!  🙏

Reply:
Works as expected (Now 😁)